products/memgraph/v0.51.0/how_to_guides/use-query-modules-provided-by-memgraph.md

```diff
@@ -51,17 +51,182 @@ loaded modules:
+take a look at installed Python files. Under Linux, the files should be located
+under `/usr/lib/memgraph/query_modules`.
+
+#### Graph Analyzer
+
+The purpose of this module is to get more insights about the stored graph. To
+illustrate functionality the following graph might be used.
+
+![](../data/graph_analyzer_graph.png)
+
+To create the graph, the following query should be executed:
+
+```opencypher
+CREATE (n1)
+CREATE (n2)
+CREATE (n3)
+CREATE (n4)
+CREATE (n5)
+CREATE (n6)
+CREATE (n7)
+CREATE (n1)-[:e]->(n5)
+CREATE (n3)-[:e]->(n5)
+CREATE (n5)-[:e]->(n2)
+CREATE (n4)-[:e]->(n2)
+CREATE (n2)-[:e]->(n6)
+CREATE (n2)-[:e]->(n7);
+```
+
+To analyze the whole graph, let's run the following query:
+
+```opencypher
+CALL graph_analyzer.analyze() YIELD *;
+```
+
+To analyze a sub-graph, relevant nodes and edges have to be collected by
+combining `MATCH` and `WITH` clauses. Once everything is collected,
+`analyze_subgraph` procedure can be called as follows:
+
+```opencypher
+MATCH (n)-[e]->(m) WITH collect(n) as nodes, collect(e) as edges
+CALL graph_analyzer.analyze_subgraph(nodes, edges) YIELD name, value
+RETURN name, value;
+```
+
+Results should be similar to the ones below.
+
+```bash
++-------------------------------------------+-------------------------+
+| name                                      | value                   |
++-------------------------------------------+-------------------------+
+| "Number of nodes"                         | "7"                     |
+| "Number of edges"                         | "6"                     |
+| "Number of bridges"                       | "6"                     |
+| "Number of articulation points"           | "2"                     |
+| "Average degree"                          | "0.8571428571428571"    |
+| "Sorted nodes degree"                     | "[(16, 4),(19, 3), ..." |
+| "Self loops"                              | "0"                     |
+| "Is bipartite"                            | "True"                  |
+| "Is planar"                               | "True"                  |
+| "Is biconnected"                          | "False"                 |
+| "Is weakly connected"                     | "True"                  |
+| "Number of weakly connected components"   | "1"                     |
+| "Is strongly connected"                   | "False"                 |
+| "Number of strongly connected components" | "7"                     |
+| "Is DAG"                                  | "True"                  |
+| "Is eulerian"                             | "False"                 |
+| "Is forest"                               | "True"                  |
+| "Is tree"                                 | "True"                  |
++-------------------------------------------+-------------------------+
+```
+
+#### Page Rank
+
+To run [Page Rank](https://en.wikipedia.org/wiki/PageRank) on top of data
+stored inside Memgraph, the following graph might be used:
```

Comment:
The `on top of` part sounds weird to me. Also, the whole sentence looks like it should be two sentences.
I would write something like this:

The Page Rank module allows you to run the Page Rank algorithm on the data stored in Memgraph.
To illustrate the functionality, the following graph will be used: